lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -40,10 +90,11 @@ sub call_rpc {
     my $before_get_rpc_response_hook = delete($req_storage->{before_get_rpc_response}) || [];
     my $after_got_rpc_response_hook  = delete($req_storage->{after_got_rpc_response})  || [];
     my $before_call_hook             = delete($req_storage->{before_call})             || [];
+    my $rpc_failure_cb               =  delete($req_storage->{rpc_failure_cb}) || 0;
```

Comment:
addition of call back

Reply:
Will `undef` be better ? or a blank sub `sub {}` ?